Dockerfile

```diff
@@ -49,6 +49,15 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
       /home/${USERNAME}/.ros && \
     chown -R $USER_UID:$USER_GID /home/${USERNAME} /opt/overlay_ws/
 
+# IMPORTANT: Optionally install Nvidia drivers for improved simulator performance with Nvidia GPUs.
+# To do this you must
+# 1. Uncomment the ENV and RUN entries below
+# 2. Replace the 'nvidia-driver-555' apt package with the Nvidia driver version on your host, e.g. nvidia-driver-535, nvidia-driver-555. Use nvidia-smi on your host to determine the driver version.
+# After rebuilding via `moveit_pro build` verify the drivers are active in your container by running `nvidia_smi` inside of `moveit_pro shell`.
+# ENV DEBIAN_FRONTEND=noninteractive
+# RUN apt update && apt install -y software-properties-common
+# RUN add-apt-repository ppa:graphics-drivers/ppa && apt update && apt upgrade -y && apt install -y nvidia-driver-555
```

Comment:
Make sure to use the Apt cache mounts on any apt installation command

Reply:
thanks Mike!